package.json

```diff
@@ -52,6 +53,7 @@
     "vue": "^2.4.0"
   },
   "devDependencies": {
+    "bundlesize": "0.17.1",
```

Comment:
It should be right after `babel-preset-es2015`.

Reply:
I expected yarn to order this and didn't check :(